app/src/main/java/org/openobservatory/ooniprobe/utils/NotificationService.java

```diff
@@ -78,56 +78,27 @@ public void sendRegistrationToServer() {
         */
 		//if device_token is null the user hasn't enabled push notifications
 		if (device_token == null) return;
-		final OrchestrateClient client = new OrchestrateClient();
-		client.set_verbosity(LogSeverity.LOG_DEBUG);
-		client.use_logcat();
-		client.set_geoip_country_path(geoip_country_path);
-		client.set_geoip_asn_path(geoip_asn_path);
-		client.set_platform(platform);
-		client.set_software_name(software_name);
-		client.set_software_version(software_version);
-		client.set_supported_tests(supported_tests);
-		client.set_network_type(network_type);
-		client.set_language(language);
-		client.set_device_token(device_token);
-		client.set_registry_url(NOTIFICATION_SERVER);
-		//client.set_available_bandwidth();
-		client.find_location(
-				(error, probe_asn, probe_cc) -> {
-					if (error.as_bool()) {
-						System.out.println(error.reason());
-						return;
-					}
-					System.out.println("ASN: " + probe_asn);
-					System.out.println("CC: " + probe_cc);
-					OrchestrateAuth auth = new OrchestrateAuth();
-					client.set_probe_asn(probe_asn);
-					client.set_probe_cc(probe_cc);
-					Error err = auth.load(auth_secret_file);
-					if (err.as_bool()) {
-						client.register_probe(
-								OrchestrateAuth.make_password(),
-								(error1, auth1) -> {
-									if (error1.as_bool()) {
-										System.out.println(error1.reason());
-										return;
-									}
-									Error err1 = auth1.dump(auth_secret_file);
-									System.out.println(
-											"Error: " + err1.reason());
-								});
-						return;
-					}
-					client.update(
-							auth,
-							(error12, auth12) -> {
-								if (error12.as_bool()) {
-									System.out.println(error12.reason());
-									return;
-								}
-								Error err12 = auth12.dump(auth_secret_file);
-								System.out.println("Error: " + err12.reason());
-							});
-				});
+		MKOrchestraClient client = new MKOrchestraClient();
+		//TODO-2.1
+		//client.setAvailableBandwidth(String value);
+		client.setDeviceToken(device_token);
+		client.setCABundlePath(ca_bundle_path);
+		client.setGeoIPCountryPath(geoip_country_path);
+		client.setGeoIPASNPath(geoip_asn_path);
+		client.setLanguage(language);
+		client.setNetworkType(network_type);
+		client.setPlatform("android");
+		//TODO what to add here?
+		//client.setProbeFamily(String value);
+		client.setProbeTimezone(TimeZone.getDefault().getDisplayName(true, TimeZone.SHORT));
```

Comment:
What string would that be?

Reply:
Should be "UTC" or "CEST" or the short version that doesn't include the Country/City

Reply:
Unless this was discussed and I missed the discussion, I would suggest to include the Country/City because it is information useful to us. Isn't it?